auth-server/test/resolvers/mutation/addressLinkConfirm.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { expect } from 'chai';
+import { AuthenticationError, ForbiddenError } from 'apollo-server';
+import 'mocha';
+
+import User from '../../../src/model/User';
+import Address from '../../../src/model/Address';
+import addressLinkStart from '../../../src/resolvers/mutation/addressLinkStart';
+import addressLinkConfirm from '../../../src/resolvers/mutation/addressLinkConfirm';
+import signup from '../../../src/resolvers/mutation/signup';
+import { Context } from '../../../src/types';
+import messages from '../../../src/utils/messages';
+
+describe('addressLinkConfirm mutation', () => {
+	let signupResult;
+	const fakectx: Context = {
+		req: {
+			headers: {}
+		},
+		res: {
+			cookie: () => {}
+		}
+	} as any;
+	const email = 'test@email.com';
+	const password = 'testpass';
+	const username = 'testuser';
+	const name = 'test name';
+
+	before(async () => {
+		signupResult = await signup(null, { email, password, username, name }, fakectx);
+		fakectx.req.headers.authorization = `Bearer ${signupResult.token}` // eslint-disable-line
+	});
+
+	after(async () => {
+		await User
+			.query()
+			.where({ id: signupResult.user.id })
+			.del();
+	});
+
+	it('should be able to confirm address link', async () => {
+		const network = 'kasuma';
+		const address = '5E9ap3x1fQ42vQ4kaD9UiSgDHZJ1WsPN64E1CMKBzAg3h7aX';
```

Comment:
Not sure this address is known to anyone. I'd go for Alice on a dev network
```suggestion
		const address = 'HNZata7iMYWmk5RvZRTiAsSDhV8366zq2YGb3tLH5Upf74F'; // Alice
```

Reply:
this will take me some time to change this. I used this address as i had access to this and could create a signature using the polkadot js extension. For using Alice address i need to know the private key and API to create a signature

Reply:
No no don't bother with that, just use Polkadot apps UI, point it to your local `--dev` node or even the hosted dev node `wss://dev-node.substrate.dev` and the UI in https://polkadot.js.org/apps/#/toolbox/sign to sign messages with Alice. That's 2min

Reply:
somehow schnorrkelVerify is returning false for the signature created by the toolbox
![Screenshot from 2020-01-29 09-21-39](https://user-images.githubusercontent.com/874046/73326654-f6106600-4278-11ea-8561-be5490e76f8d.png)
![Screenshot from 2020-01-29 09-22-40](https://user-images.githubusercontent.com/874046/73326655-f872c000-4278-11ea-9cf7-cfbfc276b1bd.png)
![Screenshot from 2020-01-29 09-22-56](https://user-images.githubusercontent.com/874046/73326656-fad51a00-4278-11ea-8a38-424480a23c25.png)